src/components/login/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes, Component } from 'react'
+import { translate } from 'react-i18next'
+
+class Login extends Component {
+	constructor (props) {
+		super(props)
+		this._handleSubmit = this.handleSubmit.bind(this)
```

Comment:
just in case of,
```jsx
this.handleSubmit = this.handleSubmit.bind(this) // removing that ugly _
```
is working fine, and you don't need to use _ anywhere